### PR TITLE
Accessibility2002

### PR DESCRIPTION
**Pull request by niusoff**

### CODE REVIEW

File: src/UXClient/Components/GroupedBarChart/GroupedBarChart.ts [hunk 1]
```diff
@@ -54,6 +54,7 @@ class GroupedBarChart extends ChartVisualizationComponent {
                 .classed("tsi-barChart", true);
             var svgSelection = targetElement.append("svg")
                 .attr("class", "tsi-barChartSVG tsi-chartSVG")
+                .attr('title', 'Bar chart')
```
Comment (darsney) [inline src/UXClient/Components/GroupedBarChart/GroupedBarChart.ts]: Strings.getString plz

File: src/UXClient/Components/DateTimePicker/DateTimePicker.ts [hunk 2]
```diff
@@ -108,6 +108,10 @@ class DateTimePicker extends ChartComponent{
         this.displayRangeErrors(rangeErrorCheck.errors);
```
Comment (darsney) [inline src/UXClient/Components/DateTimePicker/DateTimePicker.ts]: kinda was thinking that if this is created using a button, that "esc" should close it as well?  seems like a common flyout a11y pattern

Comment (darsney) [reply src/UXClient/Components/DateTimePicker/DateTimePicker.ts]: it totaly does

File: src/UXClient/Components/DateTimePicker/DateTimePicker.ts [hunk 3]
```diff
@@ -128,43 +132,50 @@ class DateTimePicker extends ChartComponent{
...
+            .classed('tsi-dateTimeSelectionPanel', true);
 
         this.timeControls = this.dateTimeSelectionPanel.append("div").classed("tsi-timeControlsContainer", true);
         this.calendar = this.dateTimeSelectionPanel.append("div").classed("tsi-calendarPicker", true);
         this.createTimezonePicker();
         var saveButtonContainer = this.dateTimeSelectionPanel.append("div").classed("tsi-saveButtonContainer", true);
         var self = this;
 
-        var onSaveOrCancel = () => {
-            this.isSettingStartTime = true;
```
Comment (darsney) [inline src/UXClient/Components/DateTimePicker/DateTimePicker.ts]: it now looks like using "enter" on save doesnt close when its a button?

File: src/UXClient/Components/LineChart/LineChart.ts [hunk 4]
```diff
@@ -1459,6 +1465,7 @@ class LineChart extends TemporalXAxisComponent {
                                 .classed("tsi-lineChart", true)
             this.svgSelection = this.targetElement.append("svg")
                                             .attr("class", "tsi-lineChartSVG tsi-chartSVG")
+                                            .attr('title', this.getString('Line chart'))
```
Comment (darsney) [inline src/UXClient/Components/LineChart/LineChart.ts]: it seems like this still doesnt really receive focus in the tab order though?

File: src/UXClient/Components/DateTimePicker/DateTimePicker.ts (hunk 3 above)
Comment (niusoff) [reply src/UXClient/Components/DateTimePicker/DateTimePicker.ts]: as far as I can tell it still does

File: src/UXClient/Components/LineChart/LineChart.ts (hunk 4 above)
Comment (niusoff) [reply src/UXClient/Components/LineChart/LineChart.ts]: as far as I can tell that's not a requirement/expected, just that the svg has a title

Comment (darsney) [reply src/UXClient/Components/LineChart/LineChart.ts]: sounds good to me

File: src/UXClient/Components/DateTimePicker/DateTimePicker.ts (hunk 3 above)
Comment (darsney) [reply src/UXClient/Components/DateTimePicker/DateTimePicker.ts]: not in DateTimeButtonRange from what i can tell, it only closes in the availabilityChart

**Pull request merged**